src/molify/utils.py

```
@@ -0,0 +1,497 @@
+def suggestions2networkx(smiles: list[str]) -> list[nx.Graph]:
+    from molify import rdkit2networkx
+
+    mols = []
+    for _smiles in smiles:
+        mol = Chem.MolFromSmiles(_smiles)
+        mol = Chem.AddHs(mol)
+        mols.append(mol)
+    return [rdkit2networkx(mol) for mol in mols]
+
+
+def draw_molecular_graph(  # noqa: C901
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `C901`)

Remove unused `noqa` directive

(RUF100)